### PR TITLE
Add Ticket card

### PR DESCRIPTION
**Pull request by panchepanevski**
Hi, 👋 in this pull request I have managed to style and add functionality to my Ticket card Element. 
The code needs refactoring, but I will do it when my app is fully designed and functional. I will discuss more with my mentors and colleges about refactoring the code and I'm planing 2 or 3 days for doing that.

My Ticket card looks like this.
ticket

Closes: #14 #15 #16 #17 #18 #19 #20 #21

### CODE REVIEW

**Comment by leonp5** [comment]
Its a big PR ;)
For the review a smaller one is easier.

File: client/src/components/Ticket/Ticket.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    height: 24px;
+    margin-top: 20px;
+  `;
+
+  return (
+    <Container>
+      <TitleWrapper>
+        <Title>{name}</Title>
+      </TitleWrapper>
+      <Edit src={editIcon}></Edit>
```
Comment (aruppert) [inline client/src/components/Ticket/Ticket.js]: You can think about alt texts, will do so soon too;)

**Comment by panchepanevski** [comment]
> db.json could be on your .gitignore.
> There you dont need to write `.db.json` because the name of the file in your case is `db.json`
> If you want to untrack the db.json now and put it on your gitignore this article helps :)
> https://dev.to/svijaykoushik/stop-tracking-and-start-ignoring-ook

Thanks, Leon I'll pack db.json direct :)

**Comment by panchepanevski** [comment]
> Its a big PR ;)
> For the review a smaller one is easier.

Yes, you are right. 👍

File: client/src/components/Ticket/Ticket.js (hunk 1 above)
Comment (panchepanevski) [reply client/src/components/Ticket/Ticket.js]: Thanks @aruppert, I totally forget about the alt attribute.

**Pull request merged**